src/components/pages/program-view/ProgramViewContainer.jsx

```diff
@@ -55,19 +82,18 @@ const ProgramViewContainer = props => {
       </TitleContainer>
 
       {/*Course Description Section*/}
-      <Container>{programData.programDesc}</Container>
+      <Container>{programData.programDescription}</Container>
```

Comment:
So here we have an example of why declaring testdata in the file is somewhat problematic. Eventually, this component will receive `programData` from `props` or from Redux State via `useSelector()`. When using Test Data of any kind, I would recommend not calling those test data variables directly and instead frame up your component to receive that variable from props or a hook or something.

Reply:
Totally valid point there too. Will be refactoring the test data into a dummy file and using props, more soon!